frontend/src/components/Playlist.js

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+
+var StanzaStore = require('../stores/StanzaStore');
+var StanzaActions = require('../actions/StanzaActions');
+
+var Sidebar = require('./Sidebar');
+var Body = require('./Body');
+
+var Playlist = React.createClass({
+	getInitialState: function(){
+		return StanzaStore.getState();
+	},
+	componentWillMount: function(){
+    StanzaStore.addChangeListener(this._onChange);
+    this.updatePlaylist();
+  },
+  componentWillUnmount: function() {
+    StanzaStore.removeChangeListener(this._onChange);
+  },
+  componentDidUpdate: function(prevProps, prevState){
+  	if (prevProps.params.emotion != this.props.params.emotion) {
+  		this.updatePlaylist();
+  	}
+ 	},
```

Comment:
Hmm, check your editor indentation it might be set on tab or something. Whatever the reason, the indentation in this file in particular is a bit all over the place.